base_link_frame_id : base_link
map_frame_id : map
loop_rate : 5
heuristic_gain : 1.0
resolution : 0.1
costmapThreshold : 90
approximationNode : 20
approximationResolution : 10
replanning_radius : 0.2